R/sankeyNetwork.R

```diff
@@ -75,7 +76,8 @@ sankeyNetwork <- function(Links, Nodes, Source, Target, Value, NodeID,
         colourScale = colourScale,
         fontsize = fontsize,
```

Comment:
to be consistent with other functions, probably should add a `fontFamily` argument.